packages/cli/package-lock.json

```diff
@@ -0,0 +1,9825 @@
+{
+  "name": "@loopback/cli",
+  "version": "1.2.2",
+  "lockfileVersion": 1,
```

Comment:
I believe we are intentionally not using package-lock files. Please remove it from the pull request.

Reply:
fixed

Reply:
I still see the package-lock file included in the PR. PTAL again.